generators/generator-botbuilder/generators/app/index.js

```diff
@@ -84,7 +84,7 @@ module.exports = class extends Generator {
             this.log(thankYouMsg);
         } else {
             const noThankYouMsg = "------------------- \n" +
-                "Bot creation has been cancelled.  \n" +
+                "Bot creation has been canceled.  \n" +
```

Comment:
In US English, canceled is typically spelled with one L. In British, Australian, and Canadian english, it's usually 2 Ls. I know this, as I looked it up just now, as the "LL" version looks more correct to me. 

... I think your fix here is correct. So thanks!